### PR TITLE
add bytesize nf-core download

### PR DESCRIPTION
**Pull request by FranBonath**
@netlify /events/2024/bytesize_nf_core_download

### CODE REVIEW

File: src/content/events/2024/bytesize_nf_core_download.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+type: talk
+startDate: '2024-07-09'
+startTime: '13:00+02:00'
+endDate: '2024-07-09'
+endTime: '13:30+02:00'
+locations:
+  - links: https://kth-se.zoom.us/j/68390542812
+---
+
+If you need to run nf-core pipelines on offline systems, it is required to first download the pipeline files before manually transfering them to your system. The `nf-core download` command will download both the pipeline code and the institutional nf-core/configs files. Learn more about `nf-core download` with Matthias Zepper ([@MatthiasZepper](https://github.com/MatthiasZepper)) in this weeks bytesize talk.
```
Comment (MatthiasZepper) [inline src/content/events/2024/bytesize_nf_core_download.md:13]:
```suggestion
Today's bytesize talk can be regarded as a continuation and supplement to of an ongoing series of talks focused on container image management in computational workflows. Over the past weeks, you already heard about the [`nextflow inspect` subcommand to list the container images required by a Nextflow pipeline](https://nf-co.re/events/2024/bytesize_singularity_containers_hpc) and learned how to best use [Singularity](https://nf-co.re/events/2024/bytesize_singularity_containers_hpc) and [Wave](https://nf-co.re/events/2024/bytesize_using_wave) for your containerization needs.

To round off the series, Matthias Zepper ([@MatthiasZepper](https://github.com/MatthiasZepper)) will showcase the use and functionality of the `nf-core download` CLI tool. It has been developed with the hope to streamline running nf-core pipelines in offline environments; a common requirement for processing sensitive genomic information, e.g. for clinical applications. The talk will cover two exemplary scenarios how to set up an offline environment and provide some guidance on the required additional configuration. Time-permitting, you may optionally receive a brief overview of the developmental roadmap and possible future features.
```

**Review by MatthiasZepper — CHANGES_REQUESTED** [review]
Thank you for adding the talk to the website! Don't you think we should put it in context with the antecedent talks in the description? I would suggest something along this line...

**Comment by FranBonath** [comment]
@MatthiasZepper  you know best what your talk will contain, so I am happy to go with any changes you suggest :)

**Review by MatthiasZepper — APPROVED** [review]
Thanks!

**Pull request merged**